.github/scripts/Create-PSQLUpdate.ps1

```diff
@@ -44,10 +44,17 @@ foreach ($item in $data) {
     Write-Output "Getting data for Org: $($item.account.id)"
     $org_installation_token = Get-InstallationToken -InstallationID $item.id -JWT (Get-Content token)
     $orgData = Get-OrgData -OrganizationID $item.account.id -InstallationToken $org_installation_token
+    if ($orgData.description -ne $null) {
+        $processed_description = $orgData.description.Replace("'", "''")
```

Comment:
@DevOptimusPrime 

Is it possible that there maybe other characters which could break this in future ? ie ["/@?]  etc ?
How if we introduce a small function to escape a few more  too

eg Got the below from the web - not sure how it works ( I am not a powershell programmer :-) ) 


function Escape-SpecialCharacters {
    [CmdletBinding()]
    param (
        [Parameter(Mandatory = $true)]
        [string]$InputString
    )

    # List of special characters to escape
    $specialCharacters = '[\^\$\.\\\|\(\)\[\]\{\}\*\+\?\']'

    # Escape special characters using regex
    $escapedString = $InputString -replace $specialCharacters, '\$&'

    return $escapedString
}

Reply:
This would be a better long term solution but its more complex for the implementation. We have to sanitize the human generated input (org names, org descriptions) for :

- Insert into the DB
- JSON blob construction (the db field is a JSON blob field type, not columns)
- Presentation to the HTML templates

I don't think one function will cover those all - it will likely need to be a module or series of functions.

Reply:
Great discussion! @furniturewalatkNIH Let’s keep a ticket open for the long term solution @DevOptimusPrime describes. I’m going to get this merged into the staging branch at least because we’ve already seen the build broken a couple times by these single quotes in the org description.